taui/src/components/custom-header-bar.js

```diff
@@ -58,7 +58,7 @@ export default class CustomHeaderBar extends Greetings {
     return (
       <header className='app-header'>
         <div className='app-header__brand'>
-          <img className='app-header__logo' src='assets/BHAlogo.png' alt='' />
+          <img className='app-header__logo' src='assets/echo_combined_logo_fullcolor.svg' alt='' />
```

Comment:
While we're in here, this would be a good opportunity to fill out the `alt` property for these images, which will help with accessibility for screenreaders.